packs/classes/alchemist.json

```diff
@@ -111,6 +111,12 @@
                 "name": "Medium Armor Expertise",
                 "uuid": "Compendium.pf2e.classfeatures.Item.Medium Armor Expertise"
             },
+            "ca55v": {
+                "img": "icons/skills/melee/hand-grip-sword-red.webp",
+                "level": 7,
+                "name": "Weapon Expertise",
+                "uuid": "Compendium.pf2e.classfeatures.Item.Weapon Expertise"
+            },
```

Comment:
Make sure to revert the ones that are staying

Reply:
Files refreshed - please check when you get a moment.